### PR TITLE
test: Acceptance tests

### PR DESCRIPTION
**Pull request by Goncalerta**
Closes https://github.com/LEIC-ES-2021-22/3LEIC04T2/issues/37

### CODE REVIEW

File: app_feup/test_report.json [hunk 1]
```diff
@@ -0,0 +1 @@
...istView\"","line":20,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:20"},"result":{"status":"passed","duration":1190000000}},{"keyword":"And ","name":"I tap the first child of type \"CheckboxListTile\" of the widget of type \"ListView\"","line":21,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:21"},"result":{"status":"passed","duration":925000000}},{"keyword":"And ","name":"I tap the back button","line":22,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:22"},"result":{"status":"passed","duration":404000000}},{"keyword":"Then ","name":"I expect the text \"Nenhuma unidade curricular\" to be present","line":23,"match":{"location":"C:\\Users\\PedroGoncaloCorreia\\Documents\\Universidade_local\\ESOF\\project-schrodinger\\app_feup\\test_driver\\features\\select_courses.feature:23"},"result":{"status":"passed","duration":24000000}}]}]}]
```
Comment (jfbaltazar) [inline app_feup/test_report.json]: I'd add this file to .gitignore and remove it.

**Review by jfbaltazar — COMMENTED** [review]
Beside that small detail on the test report file, LGTM!

File: app_feup/test_report.json (hunk 1 above)
Comment (Goncalerta) [reply app_feup/test_report.json]: I wanted to make the file available for the teacher to see, but you're right, it makes no sense for it to be here. I'm gonna bring it over to the other repository and add it to the .gitignore here.

**Review by jfbaltazar — APPROVED** [review]
With this dealt with, approved.

**Pull request merged**